### DISCUSSION TITLE
有个问题：科大讯飞能用这个工具吗

### DISCUSSION THREAD

**Discussion by hseermc** [PenUniverse/PenMods#190] [你问我答]
通过开机发现词典笔开机没有第二屏，判断是linux，有人知道这玩意能和有道一样破解吗

**Comment by Redbeanw44602** [comment]
....反正肯定不能用PenMods

**Comment by JimmyGuo12** [comment]
我两个笔都有，科大用的也是rk，但他没有设置adb，所以pENmods肯定用不了（但是科大开放了qt,所以想要破解都是小事，只是mods有没有做我就不知道了
